### PR TITLE
pkgs: init go-ethereum at 1.1.3

### PR DESCRIPTION
**Pull request by offlinehacker**
@edolstra @wkennington where i should put this? I think it should be separate package, but it breaks current design. Do we have an agreement what to do with go packages?

### CODE REVIEW

**Comment by wkennington** [comment]
I'm doing the revert so you can just put it wherever in pkgs applications. I'll be opening a PR for redesigning go-packages so don't worry about this.

**Comment by offlinehacker** [comment]
Ok

**Comment by copumpkin** [comment]
@wkennington you have a sneak peek? or can you CC me when you open the PR? I'm curious

**Comment by zimbatm** [comment]
@offlinehacker sorry it didn't get merged earlier. Can you put it somewhere in `pkgs/applications/networking` or electrum (bitcoin wallet) is in `pkgs/applications/misc` but I'm not sure it's the right categorization.

File: pkgs/top-level/go-packages.nix [hunk 1]
```diff
@@ -1108,6 +1108,28 @@ let
...
   };
 
+  go-ethereum = buildGoPackage rec {
+    rev = "v1.1.3";
+    name = "go-ethereum-${rev}";
+    goPackagePath = "github.com/ethereum/go-ethereum";
+
+    src = fetchFromGitHub {
+      owner = "ethereum";
+      repo = "go-ethereum";
```
Comment (zimbatm) [inline pkgs/top-level/go-packages.nix]: fetchFromGitHub is missing the rev attribute

**Pull request closed without merging**